azure-pipelines/build-release.yml

```diff
@@ -26,50 +26,83 @@ stages:
         strategy:
           maxParallel: 10
           matrix:
+            Assert_news:
+              python.version: '3.7'
```

Comment:
Why not 3.8?

Reply:
Previously was using python 3.7, so just retaining was done before, I don't think this would matter if 3.8 or 3.7

Reply:
I was thinking if we were adding new stuff, why wouldn't we want to use the latest python. If it was 3.7 before, this is fine.